templates/kernel/real-time/contact-us.html

```diff
@@ -327,3 +661,5 @@ <h3 class="p-heading--4">Your comments</h3>
   </section>
 
 {% endblock %}
+
+<script src="/static/js/src/contact-forms.js"></script>
```

Comment:
Ideally, we should use [versioned_static](https://github.com/search?q=repo%3Acanonical%2Fubuntu.com%20versioned_static&type=code), will accelerate loading time and allow content-cache to cache this for website failures.